Raspi/Test/serial_test.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import os
+import paho.mqtt.client as mqtt
+
+from test_data_generator import send_csv_data, send_fake_data
+
+# Arguments
+parser = argparse.ArgumentParser(description='DAS serial test script', add_help=True)
+parser.add_argument('-t', '--time', action='store', type=int, default=1, help="length of time to send data")
+parser.add_argument('-r', '--rate', action='store', type=float, default=0.5, help="rate of data in seconds")
+parser.add_argument('--host', action='store', type=str, default="localhost", help="address of the MQTT broker")
+parser.add_argument('-f', '--file', action='store', type=str, help="The csv file to replay. If not specified, makes up data.")
+parser.add_argument('-j', '--jump', action='store', type=int, default=0, help="Starts replaying from a specified time (in seconds)")
+
+def on_connect(client, userdata, flags, rc):
+    print("Connected to MQTT broker with result code " + str(rc))
+
+def run(args):
+    # Create pseudo-terminal pair
+    master, slave = os.openpty()
+
+    # Show ports
+    print("Master serial port:", os.ttyname(master))
```

Comment:
Also I get an error here.
<img width="618" alt="Screen Shot 2019-10-08 at 9 39 41 pm" src="https://user-images.githubusercontent.com/18709969/66389487-36bb4980-ea14-11e9-9fdc-7fd3803a8fb6.png">

I get a master value of 3 and a slave value of 4.

Reply:
Would you be able to have another go at this? If it still doesn't work I might need to get my hands on a mac to test on

Reply:
I still get the same thing :(
![Screen Shot 2019-12-15 at 12 05 47 am](https://user-images.githubusercontent.com/18709969/70849146-bb23c000-1ece-11ea-83aa-26feee6bd791.png)